pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -387,6 +390,12 @@ func (b *volumeBinder) AssumePodVolumes(assumedPod *v1.Pod, nodeName string) (al
 	return
 }
 
+// RevertAssumedPodVolumes will revert assumed PV and assumed PVC in cache.
+func (b *volumeBinder) RevertAssumedPodVolumes(assumedPod *v1.Pod, nodeName string) {
```

Comment:
nit: `s/will revert assumed PV and assumed PVC in cache/reverts assumed PV and PVC cache`